test-utils/state-viewer/src/main.rs

```diff
@@ -48,7 +52,7 @@ fn kv_to_state_record(key: Vec<u8>, value: Vec<u8>) -> Option<StateRecord> {
             } else {
                 let mut account = Account::try_from_slice(&value).unwrap();
                 // TODO(#1200): When dumping state, all accounts have to pay rent
-                account.storage_paid_at = 0;
+                account.storage_paid_at = genesis_height;
```

Comment:
you don't need to change it. Just keep the old value

Reply:
Oh I am stupid